tests/testthat/test-ipc.R

```diff
@@ -0,0 +1,24 @@
+test_that("Test reading data from Apache Arrow IPC", {
+
+  # Put data in Apache Arrow IPC format
+  tmpf = tempfile()
+  on.exit(unlink(tmpf))
```

Comment:
you can add in as first line  `skip_if_not_installed("arrow")`then test will not fail soft dependency arrow is not installed

Reply:
Fixed